lib/matplotlib/widgets.py

```diff
@@ -531,23 +541,22 @@ def _update(self, event):
         if self.ignore(event) or event.button != 1:
             return
 
-        if event.name == 'button_press_event' and event.inaxes == self.ax:
+        if event.name == 'button_press_event' and self.ax.contains(event)[0]:
             self.drag_active = True
             event.canvas.grab_mouse(self.ax)
 
         if not self.drag_active:
             return
 
-        elif ((event.name == 'button_release_event') or
-              (event.name == 'button_press_event' and
-               event.inaxes != self.ax)):
+        elif (event.name == 'button_release_event'
```

Comment:
```suggestion
        if (event.name == 'button_release_event'
```
Or remove the empty line above. Empty lines before elif is a bit strange...

Reply:
sure, ditto below.